src/web/components/elements/InstagramBlockComponent.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { unescapeData } from '@root/src/lib/escapeData';
+import { css } from 'emotion';
+
+const instagramoverride = css`
```

Comment:
Maybe `overrideWidth` to be more descriptive?

Reply:
Good shout.